xrally_kubernetes/env/platforms/existing.py

```diff
@@ -129,3 +129,98 @@ def _get_validation_context(self):
     def info(self):
         version = k8s_service.Kubernetes(self.platform_data).get_version()
         return {"info": version}
+
+    @classmethod
+    def create_spec_from_sys_environ(cls, sys_environ):
```

Comment:
It is quite important as first place support parsing ``$HOME/.kube/config`` file or the file specified in ``KUBECONFIG`` variable.